assignments/callbacks.js

```diff
@@ -24,27 +24,36 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, console.log);
```

Comment:
```suggestion
getLength(items, (item) => console.log(item));
```
You have supply a callback here, follow this trend down the page